CONTRIBUTING.md

```diff
@@ -10,14 +10,36 @@ To discuss anything please [contact Cornelius](mailto:cschum@suse.de).
 
 ## Contribute code
 
-To contribute code please open pull requests.
+To contribute code fork the repository and open pull requests with your changes.
 
 Ensure that rspec and rubocop pass locally before sending your PR and always that you add new changes.
 
 If your changes include important new features or bug fixes please add them to the [Master (unreleased) section fo the CHANGELOG.md](https://github.com/openSUSE/trollolo/blob/master/CHANGELOG.md#master-unreleased)
 
+### Set up
+
+Clone git repo: 
```

Comment:
I am not sure if this is needed. 🤔 It is not related to how to set Trollolo up, but with how to download a Github project.